static/js/components/CommentListMobile.jsx

```diff
@@ -0,0 +1,243 @@
+import IconButton from "@material-ui/core/IconButton";
+import CloseIcon from "@material-ui/icons/Close";
+import Typography from "@material-ui/core/Typography";
+
+import dayjs from "dayjs";
+import utc from "dayjs/plugin/utc";
+import relativeTime from "dayjs/plugin/relativeTime";
+import emoji from "emoji-dictionary";
+
+import * as sourceActions from "../ducks/source";
+import CommentList from "./CommentList";
+import styles from "./CommentList.css";
+import UserAvatar from "./UserAvatar";
+
+dayjs.extend(relativeTime);
+dayjs.extend(utc);
+
+const useStyles = makeStyles(() => ({
+  container: {
+    height: "90%",
+  },
+  commentsList: {
+    marginTop: "1rem",
+    padding: "0.25rem",
+  },
+  dialogButton: {
+    textAlign: "center",
+    margin: "1.5rem",
+  },
+}));
+
+const dialogTitleStyles = (theme) => ({
+  root: {
+    margin: 0,
+    padding: theme.spacing(2),
+  },
+  closeButton: {
+    position: "absolute",
+    right: theme.spacing(1),
+    top: theme.spacing(1),
+    color: theme.palette.grey[500],
+  },
+});
+
+const DialogTitle = withStyles(dialogTitleStyles)(
+  ({ children, classes, onClose }) => (
+    <MuiDialogTitle disableTypography className={classes.root}>
+      <Typography variant="h4">{children}</Typography>
+      {onClose ? (
+        <IconButton
+          aria-label="close"
+          className={classes.closeButton}
+          onClick={onClose}
+        >
+          <CloseIcon />
+        </IconButton>
+      ) : null}
+    </MuiDialogTitle>
+  )
+);
+
+const CommentListMobile = ({ isCandidate }) => {
+  const classes = useStyles();
+
+  const [hoverID, setHoverID] = useState(null);
+
+  const [open, setOpen] = useState(false);
+  const handleClickOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  const handleMouseHover = (id, userProfile, author) => {
+    if (
+      userProfile.roles.includes("Super admin") ||
+      userProfile.username === author
+    ) {
+      setHoverID(id);
+    }
+  };
+
+  const handleMouseLeave = () => {
+    setHoverID(null);
+  };
+
+  const dispatch = useDispatch();
+  const source = useSelector((state) => state.source);
+  const candidate = useSelector((state) => state.candidate);
+  const obj = isCandidate ? candidate : source;
+  const userProfile = useSelector((state) => state.profile);
+
+  // Color styling
+  const userColorTheme = useSelector(
+    (state) => state.profile.preferences.theme
+  );
+  const commentStyle =
+    userColorTheme === "dark" ? styles.commentDark : styles.comment;
+
+  let { comments } = obj;
+
+  comments = comments || [];
+
+  const emojiSupport = (text) =>
+    text.value.replace(/:\w+:/gi, (name) => emoji.getUnicode(name));
+
+  const recentComments = comments.slice(0, 3);
+
+  const items = recentComments.map(
+    ({
+      id,
+      author,
+      author_info,
+      created_at,
+      text,
+      attachment_name,
+      groups,
+    }) => {
+      return (
```

Comment:
```suggestion
    }) => (
```

Reply:
(with associated indentation changes following, though `prettier` will take care of that now :wink: )